ui/src/app/views/profile-page/component.ts

```diff
@@ -86,33 +100,62 @@ export class ProfilePageComponent implements OnInit {
 
   get givenNameValid(): boolean {
     if (this.workingProfile) {
-      return this.valid(this.workingProfile.givenName, 80);
+      return (ProfilePageComponent.valid(this.workingProfile.givenName, 80));
+    }
+  }
+
+  get givenNameNotEmpty(): boolean {
```

Comment:
existing issue but the function doesn't really match this return type. Should `return false;` at the end. If you let execution reach the end of a JS function without returning, that's the same as `return undefined;`